protocol/src/common/protocol.inlineCompletion.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { InlineCompletionItem, InlineCompletionContext, InlineCompletionList } from 'vscode-languageserver-types';
+import { RequestHandler } from 'vscode-jsonrpc';
+
+import { MessageDirection, ProtocolRequestType } from './messages';
+import type { TextDocumentRegistrationOptions, WorkDoneProgressOptions, StaticRegistrationOptions, WorkDoneProgressParams, TextDocumentPositionParams } from './protocol';
+
+// ---- capabilities
+
+/**
+ * Client capabilities specific to inline completions.
+ *
```

Comment:
I feel sorry but there is another thing I forgot to mention. Since this is proposed at the beginning we need to add the @proposed annotation. This is only necessary in the protocol and types package. No need on server and client since there we have to opt in anyways.

Reply:
Done.